src/main/java/com/commercetools/pingpong/model/Player.java

```diff
@@ -21,17 +23,30 @@ public Player(final String name) {
     public Player() {
     }
 
-    public void add() {
+    public void addPoint() {
         setScore++;
     }
 
-    public void sub() {
+    public void subPoint() {
         setScore--;
     }
 
-    public void reset() {
+    public void resetSet() {
         setScore = 0;
-        matchScore = 0;
+    }
+
+    public void setBeginningServe() {
+        beginningServe = true;}
```

Comment:
the bracket should go to the next line

Reply:
Ok 👍 
0827390b5ff5d3237abe982a15b51c0c618275c2